content/en/middleware.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Middleware
+status: Feedback Appreciated
+category: technology
+---
+ 
+## What it is
+ 
+Middleware is a type of computer software that provides common services and capabilities to software applications outside of what's offered by the operating system. Middleware helps developers build applications more efficiently. It makes it easier for software developers to implement communications and input/output so that they focus on building the applications. 
+ 
+ 
+## Problem it addresses 
+ 
+[Cloud native technology](/cloud_native_tech/) development brings added complexity to the development process. Developers shuffle between multiple tools, frameworks and languages also cost and time management is also a concern.  
+ 
+ 
+## How it helps 
+ 
+Developers and organizations turn to Middleware as the way to manage this complexity, also to make development quick and low-cost. Developers can build and deploy anywhere they want as it works the same thanks to Middleware running under their applications. For organizations with multi-cloud and containerized environments, Middleware is used to deploy and run apllication at [scale](/scalibilty)
```

Comment:
What do you mean with "this complexity"?  Also, not sure what you mean with "build and deploy anywhere." That sounds more like a cloud native infrastructure. Middleware exists since the late 60ies so, at its core, it should be about something else.

Reply:
Hey @CathPag, Thank you for your valuable suggestions and edits. I am working on some portions now!